Makefile

```diff
@@ -79,7 +79,7 @@ clean:
 		gen/pb-typescript/src/__generated__ \
 		gen/pb-python/sigstore_protobuf_specs/dev \
 		gen/pb-python/sigstore_protobuf_specs/io \
-		gen/pb-rust/target
+		gen/pb-rust/codegen/target
```

Comment:
@woodruffw Could you confirm this is fine? `target` seemed to be nested under codegen.

Reply:
I _believe_ that's fine, although we might want both -- `codegen` is its own sub-crate.

CC @jleightcap for confirmation.

Reply:
Thanks, added this back.